notebooks/en/pipeline_notus_instructions_preferences_legal.ipynb

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
<p>Can we use an open source model instead of OpenAI call?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>In fact, later on, we mention that an open-source model can be used, but performance will decline, what it's quite important for this tutorial to make sense.</p>